src/metabase/lib/js/metadata.cljs

```diff
@@ -24,7 +25,7 @@
 ;;; where keys are a map of String ID => metadata
 
 (defn- object-get [obj k]
-  (when obj
+  (when (and obj (js-in k obj))
```

Comment:
```suggestion
  (when (string? k)
```

Reply:
What? Did the suggestion get mangled?

Reply:
I really meant to use `(string? k)` as the test. Wouldn't that be simpler? (The following line Just Works with `obj` being `nil` and when `k` not being a property of `obj`. I think the only difference is, it doesn't blow up when `k` is not a string. Is that undesirable?

Reply:
Ah, I see. I think back when I made these changes I was trying to debug what turns out to be the `node-repl` being bad. `goog.object` doesn't work properly there, and was throwing in cases where the browser returns `nil` no problem. I think `(string? k)` suffices here, yes.